src/app/features/reciters/components/reciter-form/reciter-form.component.less

```
@@ -0,0 +1,165 @@
+    display: inline-block;
+    width: 16px;
+    height: 16px;
+    border-radius: 9999px;
+    border: 2px solid rgba(255, 255, 255, 0.4);
+    border-top-color: #fff;
+    animation: form-spinner 0.8s linear infinite;
+    margin-inline-end: 8px;
+  }
+}
+
+@keyframes slideDown {
```

Stylelint (17.3.0)

[error] Expected keyframe name "slideDown" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)